lib/core-integration/src/Test/Integration/Framework/TestData.hs

```diff
@@ -266,9 +267,14 @@ errMsg403TxTooBig n = "I had to select " ++ show n ++ " inputs to construct the\
 errMsg404MalformedTxPayload :: String
 errMsg404MalformedTxPayload = "I couldn't decode the payload that seems to be an\
     \ externally-signed transaction due to: wrongly constructed binary blob. \
-    \Make sure to send a base64-encoded binary blob, in the proper binary \
+    \Make sure to send a hex-encoded binary blob, in the proper binary \
     \format of the already-serialized transaction."
 
+errMsg404WronglyEncodedTxPayload :: String
+errMsg404WronglyEncodedTxPayload =
+    "I couldn't proceed with the payload due to wrong bytes\
```

Comment:
- Is it worth it to say `proceed with the payload`? (Honest question)
- I think `bytes encoding` should be `byte encoding`.

>  Make sure your payload

The CLI help text says `BINARY_BLOB              hex-encoded binary blob`. Are there reasons for the various terminology (I could imagine this being referred to as tx too, not sure if it is), or can we opt for fewer terms?

What about something shorter like: `I couldn't decode the payload. It doesn't appear to be hex-encoded.` or something in that direction?

Reply:
@Anviking wrote:
> What about something shorter like: "I couldn't decode the payload. It doesn't appear to be hex-encoded. or something in that direction?"

I also think this wording is preferable.

Reply:
corrected